playground/nested-deps/test-package-e/test-package-e-included/index.js

```diff
@@ -1,4 +1,4 @@
-import { testExcluded } from 'test-package-e-excluded'
+import { testExcluded } from '@vitejs/test-package-e-excluded'
```

Comment:
Instead of changing this import, we can add `test-package-e-excluded` to `optimizeDeps.exclude` instead.